src/components/Atoms/AddTaskButton/story.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Component from "./index";
+
+export default {
+  component: Component,
+  title: "Atoms,AddTaskButton",
```

Comment:
```suggestion
  title: "Atoms/AddTaskButton",
```
こうするとstorybook上でAtomsをまとめることができます